google/api_core/gapic_v1/method_async.py

```diff
@@ -32,6 +32,7 @@ def wrap_method(
     default_timeout=None,
     default_compression=None,
     client_info=client_info.DEFAULT_CLIENT_INFO,
+    kind="grpc",
```

Comment:
OK, and when we call `wrap_method` from the GAPIC, it will specify this parameter, right? So that will be in a follow-up PR in the generator repo?

Reply:
That's correct! Note to myself to update the default value to `grpc_asyncio`.

Reply:
addressed.

Reply:
Update the PR description with the right default value.